types/webpack/index.d.ts

```diff
@@ -545,7 +545,7 @@ declare namespace webpack {
             /** Give chunks created a name (chunks with equal name are merged) */
             name?: boolean | string | ((...args: any[]) => any);
             /** Assign modules to a cache group (modules from different cache groups are tried to keep in separate chunks) */
-            cacheGroups?: false | string | ((...args: any[]) => any) | RegExp | CacheGroupsOptions;
+            cacheGroups?: false | string | ((...args: any[]) => any) | RegExp | Map<string, CacheGroupsOptions>;
```

Comment:
You probably meant `{[key: string]: CacheGroupsOptions}`

Reply:
Haha same comment in the same minute 👍